sympy/sets/sets.py

```diff
@@ -55,11 +56,31 @@ class Set(Basic):
     is_ProductSet = False
     is_Union = False
     is_Intersection = None
-    is_EmptySet = None
     is_UniversalSet = None
     is_Complement = None
     is_ComplexRegion = False
 
+    @property
+    def is_empty(self):
+        """
+        Property method to check whether a set is empty.
+        Returns ``True``, ``False`` or ``None`` (if unknown).
+
+        Examples
+        ========
+
+        >>> from sympy import Interval, var
+        >>> x = var('x', real=True)
+        >>> Interval(x, x + 1).is_empty
+        False
+        """
+        return None
+
+    @property
+    @deprecated(useinstead="is_empty", issue=16946, deprecated_since_version="1.5")
```

Comment:
The alternative useinstead could be `is S.EmptySet` as you suggested

Reply:
I'll put in both. (`useinstead="is S.EmptySet or is_empty"`)

Reply:
That's good. If the deprecation refers to #16946 we should make sure that the issue has a discussion of why you might want one or the other.

Reply:
I updated the discussion in #16946. Feel free to edit.